applications/inquiry_proposal_applicants/FoxGaming.md

```diff
@@ -0,0 +1,84 @@
+
+We apply to the three-person council (fx foundation) to release the future tranchesb after we can prove the improvements.
+
+**Category your project belongs to:** NFT Marketplace
+
+
+## ***Project Description***
+
+See all hands meeting and forum:
+
+https://www.youtube.com/watch?v=CLCGdvxwM0E
+
+https://forum.functionx.io/t/foxgaming-proposal-our-preview-discussion/2757/81
+
+## ***Team profile***
+
+**More about the team:**
+
+**Thilo Colberg**, 30 years old living in germany. Owner of two financial companys and developer/owner. 
+https://de.linkedin.com/in/thilo-colberg-4b36b61b3
+
+**Jose Alberto Marquez**, 39 years old living in mexico. Chief Information Security Officer, Data Protection Officer〈 Business Model Innovation,Enterprise Architecture, IoT 
+& AI,  Researcher, Author & Speaker, SAP veteran.
+https://www.linkedin.com/in/jomarq/
+
+**Jan Matthias Colberg**, 32 years old living in kuwait city. Author and Game Full Stack Online Marketing & Digital Project Lead. 
+https://kw.linkedin.com/in/jan-matthias-colberg-98a8a7b1
+
+**Zakry Cami**, 45 years old living in Paris. Full Time Blockchain Developer and owner of an IT solution company. 
+
+Website: https://officialfoxcoin.com or https://foxchange.io
+
+Twitter: https://twitter.com/officialfoxcoin
+
+Telegram: https://t.me/foxcoinholeofficial
+
+Registered Address: Germany, 69115 Heidelberg, Langer Anger 197
+
+Contact Email: ceo@officialfoxcoin.com
+
+## ***Ecosystem Fit/Impact***
+ 
+**What problem are you solving:**
+
+Our goal is to provide some serious upgrade to function-x blockchain with a mass adoption use case!
+Every chain needs its own NFT marketplace, so here is our! FoxChange where people can sell Function X NFT & more! 
+
+**Who will benefit:**
+
+The entire FunctionX ecosystem. The most important usecase is the fee`s people have to pay in FX when they make any transaction on our marketplace or move the coins NFT´S attract a lot of people and investors.
+Are there any other similar projects in the FunctionX ecosystem: As it stands, there are currently no other projects like FoxGaming. 
+
+## ***Development Roadmap***
+
+**Milestone 1 - Kickstart the marketplace**
+Description: hire up to 10 nft artists to launch their project on FX Chain and our marketplace.
+Bitcointalkcampaign 
+banner campaign
+Milestone Tracking: After passing the vote successfully, FoxGaming will provide regular reports who we hired for the marketplace to the team.
+
+**Milestone 2 – development progress**
+Description: nft launchpad,  marketplace design 2.0, traffic on marketplace
+Milestone Tracking: After passing the vote successfully, FoxGaming will provide regular reports and submit the application for the nex tranche. 
+> ⚠️ At the inquiry stage, you will only need to fill the template up to this part. [Development status](egf-proposal-form.md#development-status) will help us understand your needs better and help us provide better assistance to you beyond financial support.
```

Comment:
also while you're add it, you can undo the deletion of the inquiry-application-template